channel/src/main/java/com/synopsys/integration/alert/channel/jira/cloud/util/JiraCloudIssueHandler.java

```diff
@@ -79,7 +82,17 @@ protected String getIssueKey(IssueResponseModel issueModel) {
     @Override
     protected IssueTrackerIssueResponseModel createResponseModel(AlertIssueOrigin alertIssueOrigin, String issueTitle, IssueOperation issueOperation, IssueResponseModel issueResponse) {
         String uiLink = JiraCallbackUtils.createUILink(issueResponse);
-        return new IssueTrackerIssueResponseModel(alertIssueOrigin, issueResponse.getKey(), uiLink, issueTitle, issueOperation);
+        return new IssueTrackerIssueResponseModel(
+            issueResponse.getKey(),
+            uiLink,
+            issueTitle,
+            issueOperation,
+            new IssueTrackerCallbackInfo(
+                alertIssueOrigin.getProviderContentKey().getProviderConfigId(),
+                alertIssueOrigin.getComponentItem().flatMap(ComponentItem::getCallbackInfo).map(ComponentItemCallbackInfo::getCallbackUrl).orElse(null),
+                alertIssueOrigin.getProviderContentKey().getSubTopicValue() // FIXME this is wrong, but this class will be deleted before 6.5.0 is released
```

Comment:
Is this for the  old Jira Cloud implementation, is that why this class is going to be removed?

Reply:
It's for the old implementation. There is a ticket to remove this class.